pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/SomaticResultUploadServiceTest.java

```diff
@@ -3,22 +3,83 @@
 import static org.junit.Assert.assertNotNull;
 
 import com.typesafe.config.Config;
+import org.broadinstitute.ddp.db.TransactionWrapper;
 import org.broadinstitute.ddp.util.ConfigManager;
+import org.broadinstitute.dsm.DbAndElasticBaseTest;
+import org.broadinstitute.dsm.db.SomaticResultUpload;
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.model.somatic.result.SomaticResultMetaData;
+import org.broadinstitute.dsm.util.DdpInstanceGroupTestUtil;
+import org.broadinstitute.dsm.util.ElasticTestUtil;
+import org.broadinstitute.dsm.util.TestParticipantUtil;
+import org.junit.AfterClass;
+import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-public class SomaticResultUploadServiceTest {
+import java.time.Duration;
+import java.time.Instant;
 
-    static SomaticResultUploadService somaticResultUploadSerivce;
+public class SomaticResultUploadServiceTest extends DbAndElasticBaseTest {
+
+    private static final DDPInstanceDao ddpInstanceDao = new DDPInstanceDao();
+    private static final String instanceName = "somatic-upload-test";
```

Comment:
Perhaps note that this name matches the configuration and thus care must be taken to change it.